index.html

```diff
@@ -2,213 +2,260 @@
-                <li class="tech">HTML</li>
-                <li class="tech">Css</li>
-              </ul>
-              <ul class="technologies">
-                <li class="tech-title">
-                  <img src="Skeleton/Rectangle.png" alt="rectangle"/>
-                  <div class="text-white">Frameworks</div>
-                </li>
-                <li class="tech">Bootstrap</li>
-                <li class="tech">Roby on Rails</li>
-                <li class="tech">RSPec</li>
-                <li class="tech">CapyBara</li>
-                <li class="tech">Selenium</li>
-              </ul>
-              <ul class="technologies">
-                <li class="tech-title">
-                  <img src="Skeleton/circle.png" alt="circle"/>
-                  <div class="text-white">Skills</div>
-                </li>
-                <li class="tech">Codekitt</li>
-                <li class="tech">Github</li>
-                <li class="tech">Codepen</li>
-                <li class="tech">Gitlab</li>
-                <li class="tech">Terminal</li>
-              </ul>
-            </div>
-        </section>
-      
-    </main>
-  </body>
+        </div>
+        <hr class="about-separater" />
+        <div>
+          <ul class="technologies">
+            <li class="tech-title font-inter">
+              <img src="./Skeleton/diamond.png" alt="diamond" />
+              <div class="text-white">Language</div>
+            </li>
+            <li class="tech">Javascript</li>
+            <li class="tech">Ruby</li>
+            <li class="tech">HTML</li>
+            <li class="tech">Css</li>
+          </ul>
+          <ul class="technologies">
+            <li class="tech-title">
+              <img src="Skeleton/Rectangle.png" alt="rectangle" />
+              <div class="text-white">Frameworks</div>
+            </li>
+            <li class="tech">Bootstrap</li>
+            <li class="tech">Roby on Rails</li>
+            <li class="tech">RSPec</li>
+            <li class="tech">CapyBara</li>
+            <li class="tech">Selenium</li>
+          </ul>
+          <ul class="technologies">
+            <li class="tech-title">
+              <img src="Skeleton/circle.png" alt="circle" />
+              <div class="text-white">Skills</div>
+            </li>
+            <li class="tech">Codekitt</li>
+            <li class="tech">Github</li>
+            <li class="tech">Codepen</li>
+            <li class="tech">Gitlab</li>
+            <li class="tech">Terminal</li>
+          </ul>
+        </div>
+      </section>
+      <section class="contact">
+        <form action="https://formspree.io/f/xlezlkyw" method="post">
+          <p class="form-detail">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+             <input type="text" placeholder="Full Name" maxlength="30" required>
+             <input type="email" placeholder="Email address" required>
```

Comment:
- [ ] Your **Full Name** and **Email** inputs are missing the bottom-border styling with have in the Figma design. Kindly add them.

- [ ] In keeping with the design, I would also recommend you remove the black borders of the message `textarea`.

  Your Implementation | Figma Design
  :---:|:---:
  ![adanze-1](https://user-images.githubusercontent.com/36542986/159699440-68f1a466-a3bb-4223-8a66-0011c22fa179.jpg) | ![adanze-figma](https://user-images.githubusercontent.com/36542986/159699445-205053e5-916e-45bd-a09b-4b67f329412b.jpg)